### ISSUE TITLE
Hex Code on mouseover

### ISSUE THREAD

**Issue by jigarius** [FortAwesome/Font-Awesome#7222]
For accessing hex-codes for any icon, we have to click on the icon and visit the full icon page. If we need more than one hex-code, we might be doing this over and over. It would be great if the hex-code and some other basic information was visible on the icon list page on mouseover.

Sorry if I posted this in the wrong place, but I believe the point is quite valid and might help many people (and help reduce a huge number of requests on the icon details pages).

**Comment by user-e** [comment]
Yes, this idea may be quite useful. I personally haven't needed the hex codes, but
_jrm1987_ and others may want to know them quickly directly from the [Icons](http://fortawesome.github.io/Font-Awesome/icons/) page.

You don't need to include the full information though, just this content:
![If this image doesn't show, click on the text box to view the image file itself where it is stored.](https://cloud.githubusercontent.com/assets/13541254/9247506/798bafae-41ab-11e5-919d-0ad506ee3e32.png)

**Comment by tagliala** [comment]
Here it is: http://fortawesome.github.io/Font-Awesome/cheatsheet/

**Comment by aronduby** [comment]
I definitely agree with the poster, but think I have a good compromise. If you could add the search from the main icons page (http://fortawesome.github.io/Font-Awesome/icons/) to the cheatsheet linked to above I think that would resolve the issue, at least for me it would.

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
